pkg/kubelet/cm/memorymanager/state/state.go

```diff
@@ -83,6 +83,7 @@ type Block struct {
 	NUMAAffinity []int           `json:"numaAffinity"`
 	Type         v1.ResourceName `json:"type"`
 	Size         uint64          `json:"size"`
+	Reused		 uint64	         `json:"reused`
```

Comment:
`Reused` stores amount of memory that was reused by app or init container from the `memoryToReuse` pool.

Reply:
True, but adding it here make sense for me as it is a container that reuses the memory. I also thought about accountability, It may be easy to check if a container reused memory by checking state file, right?

Reply:
Thanks, I corrected it. 
This is how it looks right now: 
```
	// Reused contains amount of memory common with init container's allocation
	// Reused + Size is equal to amount of container's requirement of that resource
	Reused uint64 `json:"reused"`
```